packages/@atjson/renderer-commonmark/performance/README.md

```diff
@@ -2,25 +2,23 @@
 
-This benchmark was taken on Linux 4.4 on x64 with 2 cores of Intel(R) Xeon(R) CPU @ 2.50GHz.
-
-| Function | Mean | Median | 95th Percentile | Maximum | Standard Deviation |
-|----------|------|--------|-----------------|---------|--------------------|
-| CommonMarkSource.fromRaw | 0.537ms | 0.447ms | 0.749ms | 14.923424ms | 0.461ms |
-| CommonMarkSource.convertTo(OffsetSource) | 0.350ms | 0.273ms | 0.686ms | 19.071475ms | 0.398ms |
-| CommonMarkRenderer.render | 0.121ms | 0.100ms | 0.225ms | 15.635972ms | 0.153ms |
-| Round trip | 1.066ms | 0.885ms | 2.039ms | 20.8971ms | 0.697ms |
+This benchmark was taken on Linux 4.15 on x64 with 2 cores of Intel(R) Xeon(R) CPU @ 2.30GHz.
 
+| Function                                 | Mean    | Median  | 95th Percentile | Maximum     | Standard Deviation |
+| ---------------------------------------- | ------- | ------- | --------------- | ----------- | ------------------ |
+| CommonMarkSource.fromRaw                 | 0.534ms | 0.452ms | 0.679ms         | 14.500851ms | 0.440ms            |
+| CommonMarkSource.convertTo(OffsetSource) | 0.346ms | 0.275ms | 0.683ms         | 30.790797ms | 0.381ms            |
+| CommonMarkRenderer.render                | 0.122ms | 0.100ms | 0.229ms         | 9.148015ms  | 0.138ms            |
+| Round trip                               | 1.060ms | 0.892ms | 1.996ms         | 33.880747ms | 0.659ms            |
 
 ## 🔥 Slow real-life examples
 
 The metrics below are taken from real articles written by Condé Nast editors / writers.
-
-This benchmark was taken on Linux 4.4 on x64 with 2 cores of Intel(R) Xeon(R) CPU @ 2.50GHz.
-
-| Function | Mean | Median | 95th Percentile | Maximum | Standard Deviation |
-|----------|------|--------|-----------------|---------|--------------------|
-| CommonMarkSource.fromRaw | 15.999ms | 16.813ms | 20.101ms | 24.556882ms | 3.862ms |
-| CommonMarkSource.convertTo(OffsetSource) | 53.303ms | 57.552ms | 68.802ms | 73.211224ms | 14.161ms |
-| CommonMarkRenderer.render | 45.427ms | 28.878ms | 90.374ms | 92.986452ms | 28.938ms |
-| Round trip | 114.932ms | 106.452ms | 176.815ms | 184.183862ms | 43.939ms |
+This benchmark was taken on Linux 4.15 on x64 with 2 cores of Intel(R) Xeon(R) CPU @ 2.30GHz.
+
+| Function                                 | Mean      | Median    | 95th Percentile | Maximum     | Standard Deviation |
+| ---------------------------------------- | --------- | --------- | --------------- | ----------- | ------------------ |
+| CommonMarkSource.fromRaw                 | 16.651ms  | 17.249ms  | 21.922ms        | 25.410353ms | 4.310ms            |
+| CommonMarkSource.convertTo(OffsetSource) | 55.546ms  | 59.412ms  | 70.075ms        | 70.838805ms | 13.071ms           |
+| CommonMarkRenderer.render                | 142.241ms | 74.734ms  | 264.477ms       | 489.62111ms | 106.671ms          |
+| Round trip                               | 214.641ms | 150.147ms | 356.532ms       | 577.1695ms  | 117.661ms          |
```

Comment:
Hmmm.... this is *much* slower than the last performance mark checked.

Reply:
`latest` currently has the following performance characteristics:

| Function                                 | Mean      | Median    | 95th Percentile | Maximum      | Standard Deviation |
| ---------------------------------------- | --------- | --------- | --------------- | ------------ | ------------------ |
| CommonMarkSource.fromRaw                 | 15.451ms  | 14.915ms  | 20.465ms        | 25.699598ms  | 4.285ms            |
| CommonMarkSource.convertTo(OffsetSource) | 58.132ms  | 55.576ms  | 67.408ms        | 248.79875ms  | 37.573ms           |
| CommonMarkRenderer.render                | 46.145ms  | 28.337ms  | 88.715ms        | 96.139784ms  | 29.614ms           |
| Round trip                               | 119.913ms | 100.381ms | 176.659ms       | 318.304222ms | 56.832ms           |